rust/src/ifaces/rgb21.rs

```diff
@@ -69,7 +69,7 @@ pub fn api(codex_id: CodexId) -> Api {
         },
         destructible: tiny_bmap! {
             vname!("fractions") => DestructibleApi {
-                sem_id: types.get("RGB21.NftAllocation"),
+                sem_id: types.get("RGB21.OwnedFraction"),
```

Comment:
It should be just `Nft`

```suggestion
                sem_id: types.get("RGB21.Nft"),
```